packages/page-accounts/src/Sidebar/Sidebar.tsx

```diff
@@ -249,7 +253,7 @@ export default React.memo(styled(FullSidebar)`
     > * {
       margin-bottom: 0.4rem;
 
-      &:not(:first-child) {
+      &:not(:first-sub) {
```

Comment:
? Not sure about this...

Reply:
nice catch, sorry about that, a search&replace that went wrong.
```suggestion
      &:not(:first-child) {
```